src/pipecat/services/ai_services.py

```diff
@@ -61,7 +63,10 @@ def set_model_name(self, model: str):
         self.set_core_metrics_data(MetricsData(processor=self.name, model=self._model_name))
 
     async def start(self, frame: StartFrame):
-        pass
+        if self._require_api_key and not self._api_key:
+            raise Exception(f"{self}: Error: No api_key value found.")
```

Comment:
Maybe we could make it more explicit: `No api_key provided` or `You need to pass an api_key`.